packages/app-store/office365video/api/callback.ts

```diff
@@ -104,6 +105,10 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
 
   await createOAuthAppCredential({ appId: "msteams", type: "office365_video" }, responseBody, req);
 
+  if (state?.defaultInstall) {
+    setDefaultConferencingApp(userId, "msteams");
+  }
```

Comment:
I want to avoid requiring `defaultInstall` code here per app and also requiring all new dynamic location conferencing apps to have this code too. 
It is important so that if someone adds a new conferencing app, it can be set as default during onboarding without requiring any changes in the code.

Possible Solutions:
1. Maybe there is some common function that is called for all these apps and it would make sense to move this code there. 
2. Or we simply initiate a `setDefaultConferencingApp` call from browser during onboarding after the first conferencing app is installed. 


I find Option-2 better as this default conferencing app requirement is really a requirement of onboarding only.

Reply:
> Possible Solutions:
> 
> 1. Maybe there is some common function that is called for all these apps and it would make sense to move this code there.
> 2. Or we simply initiate a `setDefaultConferencingApp` call from browser during onboarding after the first conferencing app is installed successfully. Could be done in the `onSuccess` callback
> 
> I find Option-2 better as this default conferencing app requirement is really a requirement of onboarding only.

Cannot call `setDefaultConferencingApp` directly as it has prisma. Would need to create an endpoint for it. Will check if they have some common function